internal/service/employeeaccessgrant/resource_schema.go

```diff
@@ -35,7 +35,7 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 	}
 }
 
-type TFEmployeeAccessGrantModel struct {
+type TFModel struct {
```

Comment:
interesting simplification, it does make sense to me but at the same time would opt for trying to have consistency across all our resources. Maybe we can get alignment in the technical meeting and adjust our scaffolding templates?

Reply:
sure, point added